src/components/form-textarea/form-textarea.spec.js

```diff
@@ -848,18 +848,21 @@ describe('form-textarea', () => {
 
     // Check that the internal dontResize flag is now false
     await keepalive.vm.$nextTick()
+    await new Promise(resolve => requestAnimationFrame(resolve))
```

Comment:
Use the new test utils.

Reply:
Actually not needed any more... I'll remove the await